src/app/icd/ICDManager.h

```diff
@@ -18,6 +18,7 @@
 
 #include <app/icd/ICDStateObserver.h>
```

Comment:
nit: I would be nice to have all the files related to ICD having the same naming convention
we got ICDManager.h uppercase and IcdMonitoringTable.h lowercase..

Reply:
yes, will address this in a follow up PR